packages/studio-base/src/panels/ThreeDeeRender/renderables/ImageMode/ImageMode.ts

```diff
@@ -305,18 +302,21 @@ export class ImageMode
     if (this.#removeImageTimeout == undefined) {
       this.#removeImageTimeout = setTimeout(() => {
         this.#removeImageTimeout = undefined;
-        this.imageRenderable?.dispose();
-        this.imageRenderable?.removeFromParent();
-        this.imageRenderable = undefined;
-        this.#clearCameraModel();
+        this.#removeImageRenderable();
       }, REMOVE_IMAGE_TIMEOUT_MS);
     }
+    this.#clearCameraModel();
     this.#annotations.removeAllRenderables();
     this.messageHandler.clear();
-    this.#latestImage = undefined;
     super.removeAllRenderables();
   }
 
+  #removeImageRenderable = (): void => {
+    this.imageRenderable?.dispose();
+    this.imageRenderable?.removeFromParent();
+    this.imageRenderable = undefined;
+  };
```

Comment:
nit: I don't think this actually needs to be bound
```suggestion
  #removeImageRenderable() {
    this.imageRenderable?.dispose();
    this.imageRenderable?.removeFromParent();
    this.imageRenderable = undefined;
  }
```